compiler/include/codegen.h

```diff
@@ -109,7 +109,7 @@ struct GenInfo {
   llvm::MDBuilder *mdBuilder;
   llvm::TargetMachine* targetMachine;
 
-  std::stack<LoopData> loopStack;
+  std::vector<LoopData> loopStack;
```

Comment:
Why?

Reply:
Because some of the other code iterates over the elements of the loop stack, e.g.

>      for (auto & loopData : info->loopStack) {

This is not possible with a `std::stack`.